package.json

```
@@ -17,17 +18,17 @@
     "build:docs": "nx run @refinitiv-ui/docs:build",
     "build:prod": "nx run-many --target build:prod --all",
     "clean": "npm run reset && nx run-many --target clean --parallel --all",
-    "format": "prettier --write '**/*.{js,mjs,ts,css,less,json,html,yml,md}' --ignore-unknown",
-    "start": "node cli.mjs start",
+    "format": "prettier --write '**/*.{js,mjs,cjs,ts,css,less,json,html,yml,md}' --ignore-unknown",
```

module by default across everyting including this configuration :)
```suggestion
    "format": "prettier --write '**/*.{js,cjs,ts,css,less,json,html,yml,md}' --ignore-unknown",
```